patterns/footer-centered.php

```diff
@@ -23,7 +24,17 @@
 	<div style="height:var(--wp--preset--spacing--30)" aria-hidden="true" class="wp-block-spacer"></div>
 	<!-- /wp:spacer -->
 	<!-- wp:paragraph {"align":"center","fontSize":"small"} -->
-	<p class="has-text-align-center has-small-font-size">Designed with <a href="https://wordpress.org">WordPress</a></p>
+	<p class="has-text-align-center has-small-font-size">
+		<?php
+		/* Translators: WordPress link. */
+		$wordpress_link = '<a href="' . esc_url( __( 'https://wordpress.org', 'twentytwentyfive' ) ) . '" rel="nofollow">WordPress</a>';
+		printf(
+			/* Translators: Designed with WordPress */
+			esc_html__( 'Designed with %1$s', 'twentytwentyfive' ),
+			$wordpress_link // phpcs:ignore
+		);
```

Comment:
What is the purpose of using a variable when it is only used once?

Reply:
I'd say the purpose is readability. TT4 is using it like this. We can avoid the variable if you prefer it that way.

Reply:
By avoiding it, we could also avoid the `phpcs:ignore`.

Reply:
That's true, although there's nothing wrong with using `phpcs:ignore` per se. I'll apply the chances. I consider that this way is more readable, but that may be subjective.

Thanks.